csv.go

```diff
@@ -175,7 +175,11 @@ func (cp *csvPrinter) printProbeFail(userInput userInput, streak uint) {
 		userInput.ip.String(),
 		fmt.Sprint(userInput.port),
 		fmt.Sprint(streak),
-		"-",
+		"",
```

Comment:
how about the hostname a few lines above that also is set to "-"? Do we keep consistent behavior in terms of lacking data?

Reply:
Oh yes good catch, I've made changes there too....I think empty string is better than "-" when we're lacking data

Reply:
yes, indeed